setup.py

```diff
@@ -58,6 +58,9 @@
     "fastapi>=0.68.0",
     "uvicorn[standard]>=0.14.0,<1",
     "gunicorn; platform_system != 'Windows'",
+    "dask[dataframe]>=2024.4.2",
```

Comment:
Why add these here? They are already in optional dependencies, I don't think we need any changes in required dependencies.